src/site/stages/build/process-cms-exports/schemas/common/nested-alias-string.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  $id: 'NestedAliasString',
```

Comment:
Wasn't sure if we should create something like `NestedAliasString` or `Path`, which would validate the structure of the `path` key-value pair that we'll need quite often. This currently checks only the `alias` kv pair in the `path` kv pair.

Reply:
Yeah, I think this is okay for now, but if we run into a problem later, we can re-purpose and rename this pretty easily. :+1: